README.md

```diff
@@ -362,6 +364,39 @@ in conjunction with other parameters.
   <tr><td>kibanaAdditionalYaml</td><td>string</td>
     <td>Additional configuration for Kibana yaml configuration file. Each line must be separated by a <code>\n</code> newline character e.g. <code>"server.name: \"My server\"\nkibana.defaultAppId: home"</code>. <br /><br /><strong>This is an expert level feature - It is recommended that you run your additional yaml through a <a href="http://www.yamllint.com/">linter</a> before starting a deployment.</strong></td><td><code>""</code></td></tr>
 
+  <tr><td colspan="4" style="font-size:120%"><strong>Logstash related settings</strong></td></tr>
+
+  <tr><td>logstash</td><td>string</td>
+    <td>Either <code>Yes</code> or <code>No</code> to provision a machine with Logstash installed. If you have opted to also install the X-Pack plugins using <code>xpackPlugins</code>,
+    a trial license for the <a href="https://www.elastic.co/products/stack">commercial Logstash features</a> will be applied and activated.
+    </td><td><code>No</code></td></tr>
+
+  <tr><td>vmSizeLogstash</td><td>string</td>
+    <td>Azure VM size of the Logstash instance. See <a href="https://github.com/elastic/azure-marketplace/blob/master/build/allowedValues.json">this list for supported sizes</a>.
+    <strong>Check that the size you select is <a href="https://azure.microsoft.com/en-au/regions/services/">available in the region you choose</a></strong>.
+    </td><td><code>Standard_D1</code></td></tr>
+
+  <tr><td>logstashHeapSize</td><td>integer</td>
```

Comment:
Do we need to expose this? Since logstash always deploys on a dedicated VM can we just do the right thing™?

Reply:
Do you mean the heap size? If so, I wasn't sure what might be a sensible default, beyond the default in JVM.options (which I think is 1GB?). Thoughts?

Reply:
@russcam mind opening an issue for whether we can rid of the parameter?